Docker/ProxyNginx.conf

```diff
@@ -0,0 +1,41 @@
+upstream dash-handler {
+  server 127.0.0.1:9002;
+}
+
+server {
+  listen 80;
+  server_name dash.example.com;
```

Comment:
@outcoldman  I wanted the php application as a single container, so that i can choose, if i want to proxy the connection with my webserver of favorit choice. With a linked Nginx it will be nice. We need only to edit the `docker-compose.yml`.